src/api/java/com/ldtteam/structurize/api/util/BlockPosUtil.java

```diff
@@ -211,4 +212,124 @@ public static boolean isInbetween(final BlockPos inside, final BlockPos corner1,
 
         return inside.getX() <= maxX && inside.getX() >= minX && inside.getY() <= maxY && inside.getY() >= minY && inside.getZ() <= maxZ && inside.getZ() >= minZ;
     }
+
+    /**
+     * Gets the next position to use in a circular-creeping inward snake manner. Its iterating the rectangle of corners in clockwise rotation
+     *
+     * @param start   current positions
+     * @param corner1 rectangle top left corner
+     * @param corner2 rectangle bottom left corner
+     * @return next position
+     */
+    public static BlockPos getNextPosInCircleFrom(final BlockPos start, final BlockPos corner1, final BlockPos corner2, final int ringHeight)
```

Comment:
Missing javadoc for `ringHeight`.  Also presumably `corner2` should be bottom right?

Reply:
@someaddons tststs